browser.js

```diff
@@ -113,3 +116,109 @@ document.addEventListener('DOMContentLoaded', (event) => {
   elementReady(selectors.root).then(init);
   elementReady(selectors.loginButton).then(login);
 });
+
+window.addEventListener('load', () => {
+  document.getElementById('react-root').addEventListener('contextmenu', (e) => {
+    const targetName = e.target.className;
+    if (targetName === '_ovg3g') {
+      picCover(e);
+    }
+    if (targetName === '_c2kdw') {
+      videoCover(e);
+    }
+  });
+});
+
+const removeCover = (div) => {
+  document.body.removeChild(div);
+  document.body.style.overflow = 'auto';
+};
+
+const savePicOrVideo = (src, fileType) => {
+  const title = `save ${fileType}`;
+  const options = {
+    title,
+    defaultPath: __dirname,
+  };
+  const choosePath = dialog.showSaveDialog(options);
+  if (choosePath) {
+    download(src, `${choosePath}.${fileType}`)
+      .then((done, error) => {
+        let body;
+        if (done) {
+          body = done;
+        }
+        if (error) {
+          body = done;
+        }
+        const notificationOptions = {
+          title,
+          body,
+          icon: './static/icon.png'
+        };
+        const notification = new Notification(title, notificationOptions);
```

Comment:
`Notification` is inline in electron. [you can see this example](https://github.com/hokein/electron-sample-apps/blob/master/notifications/window.js#L16)